packages/instant/tslint.json

```diff
@@ -3,6 +3,7 @@
     "rules": {
         "custom-no-magic-numbers": false,
         "semicolon": [true, "always", "ignore-bound-class-methods"],
-        "max-classes-per-file": false
+        "max-classes-per-file": false,
+        "switch-default": false
```

Comment:
I did this as part of cc2719492d72602ecd1c503cf0ecee95cd05db33

Removing the `default` in the `reducer` function allows the compiler to ensure that we explicitly switch on every `ActionType`.  It prevents us from adding a new `ActionType` and forgetting to have the reducer handle it.